front/poke/temporal/activities.ts

```diff
@@ -506,6 +507,14 @@ export async function deleteMembersActivity({
           await membership.delete(auth, t);
           await user.destroy({ transaction: t });
         }
+
+        // Delete the user's files
+        await FileModel.destroy({
```

Comment:
IIRc, this should not block and it should set to NULL the `userId` column on the `files`.

Reply:
The FK is configured with cascade. Created `FileResource.deleteAllForUser` which manually set NULL.

Reply:
Not cascade but restrict, right?

Reply:
Yeah restrict 👍

Reply:
(otherwise Sequelize default would have been SET NULL for a nullable FK)